CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ SVD files. The changes manifest in the `imxrt-ral` crate.
 
 This release also removes mention of 'stm32ral' in the API documentation.
 
+* **BREAKING** The RAL depends on `cortex-m`, version `0.7`. All `Interrupt`
+  enumerations now implement `cortex_m::interrupt::InterruptNumber`, instead
+  of `bare_metal::Nr`.
+
```

Comment:
Changelog entry is in an incorrect section. Probably introduced after rebasing. Corrected in #15.